crates/bevy_mesh/src/index.rs

```diff
@@ -0,0 +1,137 @@
+use bevy_reflect::Reflect;
+use core::iter::FusedIterator;
+use thiserror::Error;
+use wgpu::IndexFormat;
+
+/// A disjunction of four iterators. This is necessary to have a well-formed type for the output
+/// of [`super::Mesh::triangles`], which produces iterators of four different types depending on the
```

Comment:
Would be nice if these doc links were changed not to display `super::` everywhere, like they were before. Either by specifying the link separately, or if there are *tons* of these links, just importing the relevant types at module-level and adding an `#[allow(unused_imports)]` (not sure how acceptable this is generally lol)

```suggestion
/// of [`Mesh::triangles`](super::Mesh::triangles), which produces iterators of four different types depending on the
```